src/compressor/CMakeLists.txt

```diff
@@ -22,6 +22,11 @@ if(HAVE_BROTLI)
   add_subdirectory(brotli)
 endif()
 
+add_library(compressor STATIC ${compressor_srcs})
+if(HAVE_QATZIP)
+  target_link_libraries(compressor ${QATZIP_LIBRARIES})
```

Comment:
might want to add `PRIVATE` , if `QATZIP_LIBRARIES` are not supposed to be exposed by `compressor` ?